libs/designer-ui/src/lib/floatingactionmenu/index.tsx

```diff
@@ -208,9 +208,9 @@ export const FloatingActionMenu = (props: FloatingActionMenuProps): JSX.Element
   return (
     <>
       <div className="msla-dynamic-added-params-container">
-        {dynamicParameterProps.map((props) => (
-          <DynamicallyAddedParameter {...props} key={props.schemaKey} />
-        ))}
+        {dynamicParameterProps.map((props) =>
+          props.properties['x-ms-dynamically-added'] === true ? <DynamicallyAddedParameter {...props} key={props.schemaKey} /> : undefined
+        )}
```

Comment:
You should be returning null not undefined when no component needs to be rendered

Reply:
Thank you, updated